src/lib/containers/table/SynapseTable.tsx

```diff
@@ -89,7 +90,6 @@ export type SynapseTableState = {
   mapEntityIdToHeader: Dictionary<EntityHeader>
   mapUserIdToHeader: Dictionary<Partial<UserGroupHeader & UserProfile>>
   isColumnSelectionOpen: boolean
-  isUserModifiedQuery?: boolean //flag to signal that the selection criterial has been defined by user and if no records are returned do not hide the table
```

Comment:
This was trying to track if a user action caused it to result in zero rows, but I could not find a way to do this.  Maybe if you quickly select facet values (across facet keys)?  Facet values that would result in zero rows are not returned by the service.